src/openafval/trash/models.py

```diff
@@ -0,0 +1,263 @@
+
+
+class EntityObjectManagement(models.Model):
+    id: int
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the entity linked to BAG object."),
+        max_length=255,
+        unique=True,
+    )
+    entity = models.ForeignKey(Entity, on_delete=models.CASCADE)
+    bag_object = models.ForeignKey(BagObject, on_delete=models.CASCADE)
+    start_date = models.DateField(
+        _("start date"),
+        help_text=_(
+            "The start date of when the entity (subject) is in use of the BAG object."
+        ),
+        null=True,
+    )
+    end_date = models.DateField(
+        _("end date"),
+        help_text=_(
+            "The date of when the entity (subject) is no longer in use of the "
+            "BAG object."
+        ),
+        null=True,
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("entity object management")
+        verbose_name_plural = _("entity object managements")
+
+    def __str__(self):
+        return f"{self.entity} - {self.bag_object}"
+
+
+class Fraction(models.Model):
+    id: int
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the Fraction."),
+        max_length=255,
+        unique=True,
+    )
+    description = models.TextField(
+        _("description"),
+        help_text=_("The description of the Fraction."),
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("faction")
+        verbose_name_plural = _("fractions")
+
+    def __str__(self):
+        return self.description
+
+
+class ContainerType(models.Model):
+    id: int
+    fraction = models.ForeignKey(
+        Fraction,
+        verbose_name=_("fraction"),
+        help_text=_("The fraction of the container."),
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    type = models.CharField(
+        _("type"),
+        max_length=120,
+        help_text=_("The type of the container."),
+        unique=True,
+    )
+    description = models.TextField(
+        _("description"),
+        help_text=_("The description of the container."),
+        blank=True,
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("container type")
+        verbose_name_plural = _("container types")
+
+    def __str__(self):
+        return f"{self.type} - {self.description}"
+
+
+class Container(models.Model):
+    id: int
+    type = models.ForeignKey(
+        ContainerType,
+        verbose_name=_("type"),
+        help_text=_("The type of the container."),
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the Container."),
+        max_length=255,
+        unique=True,
+    )
+    is_collection_container = models.BooleanField(
+        _("is collection container"),
+        help_text=_("Whether the container has a collection container."),
```

Comment:
The field name is `is_collection_container` but the help text says "has a collection container". I think it should be "is".